Symfony/CS/Tests/Fixer/Contrib/NoUselessReturnFixerTest.php

```diff
@@ -34,6 +34,44 @@ public function provideFixCases()
         return array(
             array(
                 '<?php
+                    function bar($baz)
+                    {
+                        if ($baz)
+                            return $this->baz();
+                        else
```

Comment:
i still think having access to the AST might help us make changes like that, in more general